mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -447,55 +288,13 @@ public Process ()
 			}
 		}
 
-		[MonoLimitation ("Under Unix, only root is allowed to raise the priority.")]
-		[DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
-		[MonitoringDescription ("The relative process priority.")]
-		public ProcessPriorityClass PriorityClass {
```

Comment:
The setter in referencesource does some unnecessary checks for Win2k that can be removed.